misc/default_config.json

```diff
@@ -40,65 +41,76 @@
 	"server_local" : true,
 	"server_verbose" : true,
 	"show_fps" : false,
+	"show_key_icons" : false,
 	"show_messages" : true,
-	"show_key_icons": false,
 	"show_tracked_variables" : true,
 	"sound_volume" : 75.0,
-	"joystick_deadzone": 5.0,
-	"t_down" :
+	"t_down" : 
 	[
-		[ "kDown" ],
-		[ "kS" ]
+		[ "kS" ],
+		[ "" ]
 	],
-	"t_exit" :
+	"t_exit" : 
 	[
-		[ "kEscape" ]
+		[ "" ],
+		[ "" ]
 	],
-	"t_focus" :
+	"t_focus" : 
 	[
 		[ "kLShift" ],
 		[ "bXButton1" ]
 	],
-	"t_force_restart" :
+	"t_force_restart" : 
 	[
-		[ "kR" ],
-		[ "bXButton2" ]
+		[ "" ],
+		[ "" ]
 	],
-	"t_restart" :
+	"t_replay" : 
 	[
-		[ "kReturn" ],
-		[ "bMiddle" ]
+		[ "" ],
+		[ "" ]
 	],

```

Comment:
Ditto, should these changes be committed? Don't we want kReturn and bMiddle to be defaults?

Reply:
That must have been an oversight on my end, I will recheck the .json